### ISSUE TITLE
When ALL Search parameters are included we get zero package results ... :-(

### ISSUE THREAD

**Issue by nelsonic** [numo-labs/lambda-ne-classic-package-provider#44]
Given the following SNS Message: 
https://github.com/numo-labs/lambda-ne-classic-package-provider/blob/2928b112db04a6167b83a770758f177d8eb491bd/test/fixtures/complete_sns_event.json#L12
We only get 3 results.
But given a departure date 2 weeks in the future we almost *never* get results back... :-(
I propose we *temporarily* remove the `departureDate` until we can confirm with Jesper how to send them a *range* of dates...

**Comment by jruts** [comment]
@nelsonic would it be an option to set the default date a bit more in the future. 
People usually book or search for a holiday a few months in advance.
By setting the default date 2 weeks from now we are basically targeting the 'last minute' holidays people. In that regard it does make sense that there is not that much availability anymore.

We could be smart here and detect there are no results for a date in the near future and show 'last minute deals' as the 'extra packages/tiles'.

Any thoughts on this?

**Comment by nelsonic** [comment]
@jruts agree. We should be setting the date a few _months_ in the future. Might be worth asking Rasmus/Pascal what the best default date would be...?
